### DISCUSSION TITLE
4.0.5 settings.json location

### DISCUSSION THREAD

**Discussion by 00mpy** [transmission/transmission#6808] [Q&A]
### What is the issue?

Can anyone point me in the right direction for the location of the settings.json file for v4.0.5 that was built from the repo on ubutnu server?

I am not able to access the web interface.

### Which application of Transmission?

transmission-daemon

### Which version of Transmission?

4.0.5

**Comment by killemov** [comment]
https://askubuntu.com/questions/276649/where-is-the-settings-json-file-stored-for-transmission

**Comment by 00mpy** [comment]
after much playing and trying differnet install methods
i installed ubuntu 24.04 LTS and installed via nala (apt)
i still found 2 locations for the settings

~/.config/transmission-daemon/setting.json & /etc/transmission-daemon/settings.json

when started via service transmission ignores ~/.config/transmission-daemon/setting.json
instead uses /etc/transmission-daemon/settings.json

wouldn't this be better as a symlink??


regarding my OP, i changed install method & OS version to get what i needed
i never found the settings.json file when transmission was built from repo/source
